pkg/proxy/apis/config/v1alpha1/defaults_test.go

```diff
@@ -31,6 +31,7 @@ func TestDefaultsKubeProxyConfiguration(t *testing.T) {
 	oomScore := int32(-999)
 	ctMaxPerCore := int32(32768)
 	ctMin := int32(131072)
+	disableNodePort := true
```

Comment:
I think that what you don't need to declare a new variable, the name is also confusing since you want a poitner to `true` just to compare , right?

just import `utilpointer "k8s.io/utils/pointer"` and use `utilpointer.BoolPtr(true)`

Reply:
I agree, Maybe `masqBit` also don't need to declare a new variable? 
```
IPTables: kubeproxyconfigv1alpha1.KubeProxyIPTablesConfiguration{
	MasqueradeBit:             utilpointer.Int32(14),
	MasqueradeAll:             false,
	DisableLocalhostNodePorts: utilpointer.Bool(false),
	SyncPeriod:                metav1.Duration{Duration: 30 * time.Second},
	MinSyncPeriod:             metav1.Duration{Duration: 1 * time.Second},
}

```